webpack.config.js

```diff
@@ -10,15 +10,19 @@
 
 "use strict";
 var path = require('path');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = {
     cache: true,
-    entry: "./js/core/coreApi.js",
+    entry: {
+        core: "./js/core/coreApi.js",
+        styles: "./resources/styles/styles.js"

```

Comment:
You shouldn't create a separate entry, simply import the styles inside coreApi.js